static/etc/lsb-release

```diff
@@ -0,0 +1,4 @@
+DISTRIB_ID=Ubuntu
+DISTRIB_RELEASE=18.04
+DISTRIB_CODENAME=bionic
+DISTRIB_DESCRIPTION="Ubuntu Core 18"
```

Comment:
The main reason for that is that /etc/update-motd.d/00-header references it (as per what's in our current base-files). It only falls back to using lsb_release, the command, when /etc/lsb-release is not present. It does not use os-release by default, and I would prefer not to fallback to running a command on every login without need.

I could modify /etc/update-motd.d/00-header to link in /etc/os-release instead if that's preferable. But I suppose if lsb-release is deprecated, maybe we should modify the base-files package not to use it in the motd in the first place (and possibly backport it to bionic)? This way we wouldn't have to hack anything.

I'm leaving the decision up to you Steve. For now I'm leaving it as-is because I anyway think we shouldn't leave deprecated files around with deprecated contents in them. I mean, we might recommend using os-release, but lsb-release shouldn't be left outdated anyway (outdated as in not mentioning that the distro is a core 18 system, as what we have in os-release). But maybe I'm overthinking this!